src/shared/components/app-bar/AppBar.scss

```diff
@@ -8,7 +8,9 @@
     'top top'
     'main right';
   align-items: center;
-  padding: 8px 20px;
+  padding: 8px 0px;
```

Comment:
this should be changed either to `padding-top`, or to `padding: 8px $page-right-padding 0 $page-left-padding`;